gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -99,20 +99,21 @@ def run(argv=None):
 
   temp_folder = google_cloud_options.temp_location or tempfile.mkdtemp()
   timestamp_str = datetime.now().strftime('%Y%m%d_%H%M%S')
+  _, _, table_id = bigquery_util.parse_table_reference(known_args.input_table)
```

Comment:
For VCF to BQ, we actually use the job_name: https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/vcf_to_bq.py#L166 ... i think we should do the same here as you can technically have the same problem with the table ID. So, I think we need a common lib to use for both here and vcf_to_bq (this lib would also include the `_is_direct_runner` function). For now, feel free to copy the code here for using job_name here and then add a TODO to refactor it later.

Reply:
Thanks Asha! Done. Yes, the job name is much better than the table id since the job name must be unique, or an error occurs. It is much better to raise an error than to provide a wrong file.